pg_backup_api/pg_backup_api/run.py

```diff
@@ -126,7 +126,10 @@ def main():
     p_ops.set_defaults(func=recovery_operation)
 
     args = p.parse_args()
-    return args.func(args)
+    if hasattr(args, "func") is False:
```

Comment:
I was curious by your solution as I did not understood it at first.
You could also add this before returning:
`p.set_defaults(func=lambda x: p.print_help())`
It will handle the if else test for you

Reply:
Actually it needs to be set before parsing `p`